l1-contracts/bootstrap.sh

```diff
@@ -15,12 +15,13 @@ if [ -n "$CMD" ]; then
   fi
 fi
 
-# Install foundry.
-. ./scripts/install_foundry.sh
 
 # Attempt to just pull artefacts from CI and exit on success.
 [ -n "${USE_CACHE:-}" ] && ./bootstrap_cache.sh && exit
 
+# Install foundry.
+. ./scripts/install_foundry.sh
```

Comment:
I think this will still do the installation, doesn't look like the script checks if it's already installed.

Reply:
The idea is skipping it on `fast` bootstrap only, since it exits the script

Reply:
ah yes, I missed the `&& exit` part